### PR TITLE
Support .NET Core 2.2

### PR DESCRIPTION
**Pull request by tmds**
This is for merging dev_22 branch to master.

These changes have already been reviewed. We can merge this when we want.

### CODE REVIEW

File: 2.2/build/test/precompiled/console.runtimeconfig.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "runtimeOptions": {
+    "tfm": "netcoreapp2.2",
+    "framework": {
+      "name": "Microsoft.NETCore.App",
+      "version": "2.2.0-preview2-26905-02"
```
[the remark this answers is not in the document]
Comment (aslicerh) [reply 2.2/build/test/precompiled/console.runtimeconfig.json]: Yeah, but the tests didn't finish (and therefore confirm the changes) until after you'd left. I meant to make a PR last night with the changes, but got distracted by all the other stuff going on.

[the remark this answers is not in the document]
Comment (tmds) [reply 2.2/build/test/precompiled/console.runtimeconfig.json]: I've pushed a commit that removes the version suffixes.

[the remark this answers is not in the document]
Comment (aslicerh) [reply 2.2/build/test/precompiled/console.runtimeconfig.json]: I saw. I think everything looks good at this point. Thanks for making the update!

**Comment by aslicerh** [comment]
[cibuild]

**Pull request merged**